src/index.html

```diff
@@ -2,17 +2,90 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
```

Comment:
Keep your attributes correctly formatted. If the HTML-element has long attribute values or a number of attributes is more than 2, start each one, including the first, on a new line with 2-space indentation related to the tag.